src/ray/core_worker/core_worker.h

```diff
@@ -1849,6 +1858,16 @@ class CoreWorker : public rpc::CoreWorkerServiceHandler {
   uint32_t pid_;
 
   absl::flat_hash_set<ObjectID> deleted_generator_ids_;
+
+  /// Serialized runtime info env are cached.
+  mutable std::mutex runtime_env_serialization_mutex_;
+  /// Maps serialized runtime env to **immutable** deserialized protobuf.
+  mutable boost::compute::detail::lru_cache<std::string,
```

Comment:
Can we temporarily change this back to std::unordered_map so that this PR compiles and we can see the real perf impact on the release test. I'm curious whether the release test will see the same perf improvement as what you observed locally.

Reply:
Sure, release run launched: https://buildkite.com/ray-project/release/builds/27207

Reply:
Is this with the modification of release test to add env var runtime env?